content/data/home.yml

```diff
@@ -0,0 +1,33 @@
+- title: Connect storage to repo
+  description: Keep large data and model files alongside code and share via your cloud storage.
+  terminal: |
+    $ dvc add cats-dogs
+    
+    $ dvc remote add storage s3://bucket/dvc-cache
+    
+    $ dvc push
+    5000 files pushed
+- title: Configure steps as you go
+  description: Declare dependencies and outputs at each step to build reproducible end-to-end pipelines.
+  terminal: |
+    $ dvc exp run
+    "data/data.xml.dvc" didn't change, skipping^500
+    Stage "prepare" didn't change, skipping^500
+    Stage "featurize" didn't change, skipping^500
+    Running stage "train":
+    > python src/train.py data/features model.pkl
+- title: Track experiments in Git
+  description: Track experiments in your repo, compare results and restore entire experiment states cross-team.
+  terminal: |
+    $ dvc exp show
+    ───────────────────────────────────────────────────────────────────────────
+    <span class="text-black"><span class="bg-stone-100"> Experiment               </span> <span class="bg-orange-200"> roc_auc.train </span> <span class="bg-orange-200"> roc_auc.test </span> <span class="bg-cyan-200"> train.min_split </span></span>
+    ───────────────────────────────────────────────────────────────────────────
+     <b>workspace                        0.96464        0.93989   0.04</b>
+     <b>main                             0.98667        0.94602   0.01</b>
+     ├── 0417ff0 [fuggy-zack]         0.98667        0.94602   0.01
+     ├── cb5c648 [fishy-mene]         0.97652        0.94309   0.02
+     ├── c0dca21 [atrip-teds]         0.96957         0.9407   0.03
+     ├── 3d2c9c7 [brisk-aged]         0.96464        0.93989   0.04
+     └── 6e910c4 [chirk-main]         0.96193        0.93979   0.05
+    ───────────────────────────────────────────────────────────────────────────
```

Comment:
_much_ easier to work with than JSON for this purpose!

Reply:
yeah, looks great!